coderd/cryptokeys/dbkeycache.go

```diff
@@ -61,18 +76,39 @@ func NewDBCache(logger slog.Logger, db database.Store, feature database.CryptoKe
 	return d
 }
 
-// Verifying returns the CryptoKey with the given sequence number, provided that
+func (d *dbCache) EncryptingKey(ctx context.Context) (id string, key interface{}, err error) {
+	return d.latest(ctx)
+}
+
+func (d *dbCache) DecryptingKey(ctx context.Context, id string) (key interface{}, err error) {
+	return d.sequence(ctx, id)
+}
```

Comment:
`Encrypting` only returns the latest, and `Decrypting` returns the key by ID **iff** it's still valid?

Can we add comments to these functions, and to the Signing ones below to mention this behavior?

Reply:
Added to the interfaces